contracts/base/Escrow.sol

```diff
@@ -181,10 +181,11 @@ contract Escrow is EscrowInterface, TInitializable, BaseEscrowDapp {
     function claimTokens(address recipient) external {
         require(getLoan().status != TellerCommon.LoanStatus.Active, "LOAN_ACTIVE");
         if (getLoan().liquidated) {
+            // require liquidated
             require(recipient != getBorrower(), "RECIPIENT_CANNOT_BE_BORROWER");
             require(msg.sender == address(loans), "CALLER_MUST_BE_LOANS");
         } else {
-            require(recipient == getBorrower(), "RECIPIENT_MUST_BE_BORROWER");
+            require(recipient == getBorrower(), "RECIPIENT_MUST_BE_BORROWER"); // remove
         }
```

Comment:
1. Should have `onlyOwner` modifier so only the borrower can claim it.
1. Remove the `recipient` parameter and the require statement for it.
1. Remove check for caller being the loans contract.
1. Require the loan is `Closed` and `liquidated`

Reply:
Changed